things/database.py

```diff
@@ -727,28 +727,22 @@ def convert_thingstime_sql_expression_to_isotime(sql_expression: str) -> str:
     Parameters
     ----------
     sql_expression : str
-        A sql expression pointing to a "Things time" integer in
-        format hhhhhmmmmmm00000000000000000000, in binary.
-        
+        A sql expression pointing to a "Things time" integer in format hhhhhmmmmmm00000000000000000000, in binary.
```

Comment:
Try to keep docstrings less wide; usually we do 79 characters at most, but less is sometimes even more readable. What you had before is more preferable.